docs/components/connectors/connector-sdk.md

```diff
@@ -12,10 +12,12 @@ The **Connector SDK** is in developer preview and subject to breaking changes. U
 :::
 
 The **Connector SDK** allows you to [develop custom Connectors](#creating-a-custom-connector)
-using Java code. You can focus on the logic of the Connector, test it locally, and
-reuse its runtime logic in multiple environments. The SDK achieves this by abstracting from
+using Java code.
```

Comment:
Theoretically it's Java or Kotlin but Kotlin was never tried out yet. No need to update it now until we have a realistic use-case, just a not to self, @sbuettner, and @bastiankoerber: something that we should take care in the future. I think, the Community will like it.